raiden/tests/fixtures/blockchain.py

```diff
@@ -104,7 +104,8 @@ def ensure_dag_is_generated(request, blockchain_type, dagpath):
 
     if blockchain_type == 'geth' and missing_dag:
         dag_folder = os.path.dirname(dagpath)
-        os.makedirs(dag_folder)
+        if not os.path.exists(dag_folder):
+            os.makedirs(dag_folder)
```

Comment:
You can use `os.makedirs(dag_folder, exist_ok=True)`, which will do nothing if directory exists, but is just a small detail.